03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,19 @@
+1. Quickly explain how middleware is used in an Express application.
+
```

Comment:
Did you have an answer for this quiz question? It doesn’t block assignment approval but good to make sure you have an understanding